### DISCUSSION TITLE
artisan command to get livewire version

### DISCUSSION THREAD

**Discussion by joostvanhoof** [livewire/livewire#1148] [4. General]
Maybe this is my inability to find package versions, but I just created a Github issue for Livewire where the issue template asks for the Livewire version.

The same as for finding Laravel's version I went to the terminal only to return here with sadness and confusion.

How about... `php artisan livewire --version`?

Running that command currently actually returns Laravel's version, so maybe `php artisan livewire:version` is better.

**Comment by devcircus** [comment]
I've always just used composer show company/package. So:
```composer show livewire/livewire```

**Reply by joostvanhoof** [reply]
Thanks! There could still be a `version` command like Laravel has but I'll mark this as an answer as that would be a nice to have.
